src/main/pages/che-7/overview/con_persistent-volume-configuration.adoc

```diff
@@ -1,8 +1,136 @@
+
+User-defined PVCs are created with few modifications:
+
+- they are provisioned with generated names to guarantee that it is not conflicting with other PVCs in namespace;
+
+- subpaths of mount volumes that reference user-defined PVCs are prefixed with `{workspace id}/{PVC name}`.
+It is done to have the same data structure on PV on different PVC strategies(link:#how-the-subpaths-are-used[more]);
+
+[id="common-pvc-strategy"]
+= Common PVC Strategy
+
+[id="how-the-common-pvc-strategy-works"]
+== How the common PVC strategy works
+
+All workspaces (within one {k8s-namespace}) use the same PVC to store data in their declared volumes (projects and workspace logs by default and whatever additional volumes that a user can define.)
+
+User-defined PVCs are ignored and volumes that reference PVCs are replaced with volume that references common PVC.
+The corresponding containers volume mounts are relinked to common volume and subpaths are prefixed with `'{workspaceId}/{originalPVCName}'` (link:#how-the-subpaths-are-used[more]).
+
+User-defined PVC name is used as Che Volume name. It means that if Machine is configured to use Che Volume with the same name as user-defined
+PVC has then they will use the same shared folder in common PVC.
+
+When a workspace is deleted, a corresponding subdirectory (`${ws-id}`) is deleted in the PV directory.
+
+[id="restrictions-on-using-common-pvc-strategy"]
+== Restrictions on using the common PVC strategy
+
+When the `common` strategy is used and a workspace PVC access mode is ReadWriteOnce (RWO), only one {admin-context} node can simultaneously use the PVC.
+If there are several nodes, you can use the `common` strategy, but:
+- the workspace PVC access mode must be reconfigured to ReadWriteMany (RWM), so multiple nodes can use this PVC simultaneously.
+- the only one workspace in the same namespace may be running. See how to configure link:#workspace-namespaces-configuration_che-workspace-configuration[namespace strategy] and link:#TODO[running workspace count]
+
+[id="per-workspace-pvc-strategy"]
+= Per workspace PVC strategy
+
+[id="how-the-per-workspace-pvc-strategy-works"]
+== How the per-workspace PVC strategy works
+
+The `per-workspace` strategy works similarly to the `common` PVC strategy. The only difference is that all workspace volumes (but not all workspaces) use the same PVC to store data (projects and workspace logs by default and any additional volumes that a user can define).
+
+
+[id="how-the-subpaths-are-used"]
+= How the subpaths are used in PVCs
+
+----
+/pv0001
+  /workspaceid1
+  /workspaceid2
+  /workspaceidn
+    /che-logs
+    /projects
+    /<volume1>
+    /<volume2>
+    /<User-defined PVC name 1 | volume 3>
+    ...
+----
+
+Volumes can be anything that a user defines as volumes for workspace machines.  The volume name is equal to the directory name in `${PV}/${ws-id}`.
+
+
+[id="configuring-a-needed-workspace-strategy"]
```

Comment:
It is the right place to put such info? Maybe it would be better to move somewhere in the `Administration Guide` section along with the info about configuring namespaces strategies?
@rkratky WDYT?